resources/guis.yml

```diff
@@ -0,0 +1,244 @@
+---
+guis:
+- name: SourceTree
+  url: https://www.sourcetreeapp.com/
+  image_tag: guis/sourcetree@2x.png
+  platform_classes:
+  - mac
+  - windows
+  platforms:
+  - Mac
+  - Windows
```

Comment:
Seems like we could probably generate `platforms` from `platform_classes` (or vice versa) rather than require anyone updating the yaml to keep them in sync.

Reply:
I thought about that yes (I think it's mentioned in one of the commit messages). I can do a small improvement of course

Reply:
just pushed this improvement